# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../icecream/no-cc-links.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- icecream/client/Makefile.am.vanilla	2007-06-12 04:52:25.669560466 +0200
+++ icecream/client/Makefile.am	2007-06-12 04:53:03.567720159 +0200
@@ -14,10 +14,6 @@
 
 install-exec-local:
	$(mkinstalldirs) $(DESTDIR)$(bindir)
-	for link in g++ gcc c++ cc; do \
-	  rm -f $(DESTDIR)$(bindir)/$$link ;\
-	  $(LN_S) icecc $(DESTDIR)$(bindir)/$$link ;\
-	done
 
 uninstall-local:
 	for link in g++ gcc c++ cc; do \
